JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+msgstr "Gewichten"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Type"
+msgstr "Type"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model-averaged"
+msgstr "Modelgemiddeld"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional"
+msgstr "Conditioneel"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Show priors"
+msgstr "Toon priors"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional models only"
+msgstr "Alleen de geconditioneerde modellen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model number"
+msgstr "Model nummer"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "MCMC Diagnostics"
+msgstr "MCMC Diagnostiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Overview"
+msgstr "Overzicht"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Include theta"
+msgstr "Includeer theta"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Plot"
+msgstr "Grafiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Trace"
+msgstr "Spoor"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Autocorrelation"
+msgstr "Autocorrelatie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Posterior samples densities"
+msgstr "Posterior steekproefdichtheid"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform estimates"
+msgstr "Transformeer schattingen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform correlations"
+msgstr "Getransformeerde correlatie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Cohen's d"
+msgstr "Cohen's d"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Fisher's z"
+msgstr "Fisher's z"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Bridge sampling iterations:"
+msgstr "Bridge sampling iteraties:"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimation settings (MCMC)"
+msgstr "Schattingsinstellingen (MCMC)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Adaptation"
+msgstr "Aanpassing"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Burnin"
+msgstr "Opwarming"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Iterations"
+msgstr "Iteraties"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Chains"
+msgstr "Ketens"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Thin"
+msgstr "Dun"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Autofit"
+msgstr "Autopassing"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Maximum fitting time"
+msgstr "Maximale pastijd"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "hours"
+msgstr "uren"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "minutes"
+msgstr "minuten"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "seconds"
+msgstr "seconden"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Target margin of error"
+msgstr "Doelwit foutmarge"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Exclude models"
+msgstr "Sluit modellen uit"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "error % >"
+msgstr "fout % >"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "R-hat >"
+msgstr "R-hat >"
```

Comment:
R-dakje? Of is dat erg raar

Reply:
Als je er een sterke mening over hebt, dan verander ik R-hoed in R-dakje overal. Op dit moment noem ik het overal R-hoed

Reply:
Okay overal R-dakje